sky/shell/platform/ios/framework/Headers/FlutterAppDelegate.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERAPPDELEGATE_H_
```

Comment:
The rest of the public headers in this directory have these include guards.  It's possible we should remove them from all the headers, but I'd be inclined to make that change separately.